rfcs/accepted/0001-cli-atomic-taxonomy.md

```diff
@@ -0,0 +1,106 @@
+- Feature Name: Taxonomy for Atomic in the Coveo CLI
+- Start Date: 2023-03-08
+
+# Summary
+
+[summary]: #summary
+
+Create a one-stop-shop topic for CLI user that uses Atomic:
+
+- `atomic:deploy` alias `ui:deploy`
+- `atomic:init` would either 'run' `ui:create:atomic` or start a blank lib project `npm init @coveo/atomic-component-project`, depending on user input.
+- `atomic:cmp` would either `npm init @coveo/atomic-component` or `npm init @coveo/atomic-result-component`
```

Comment:
I would prefer if we didn't try to shorten the word "component", and made this command use the full word: `atomic:component`. I don't feel like it's very obvious that `cmp` means "component".

Reply:
Alright, let's do both :D `atomic:component` and its alias `atomic:cmp`